portality/templates/doaj/index.html

```diff
@@ -45,26 +45,34 @@ <h1>Find open access journals & articles.</h1>
             <section class="row mobile-scrollable-row">
                 <h2 class="sr-only">DOAJ in numbers</h2>
                 <p class="col-md-2 stat">
-
-                        <span class="stat__number">{{ statistics.countries }}</span>
-                        <span class="stat__description">countries represented</span>
-
+                    <span class="stat__number">{{ statistics.countries }}</span>
+                    <span class="stat__description">countries represented</span>
                 </p>
                 <p class="col-md-2 stat">
+                    {# link to query for journals published in 2019, need to find a better way to do this #}
+                    <a href='/search/journals?source=%7B%22query%22%3A%7B%22filtered%22%3A%7B%22filter%22%3A%7B%22bool%22%3A%7B%22must%22%3A%5B%7B%22range%22%3A%7B%22created_date%22%3A%7B%22gte%22%3A%221546300800000%22%7D%7D%7D%5D%7D%7D%2C%22query%22%3A%7B%22match_all%22%3A%7B%7D%7D%7D%7D%2C%22size%22%3A50%2C%22sort%22%3A%5B%7B%22created_date%22%3A%7B%22order%22%3A%22desc%22%7D%7D%5D%7D'>
                         <span class="stat__number">{{ statistics.new_journals }}</span>
                         <span class="stat__description">new journals</span>
+                    </a>
                 </p>
                 <p class="col-md-2 stat">
+                    {# link to query for journals w/out APCs #}
```

Comment:
```suggestion
                    {# TODO: link to query for journals w/out APCs #}
```
As above, we don't want to hard-code the path here in the template. Todo added for fixup later.